bundle/manifests/operator.shipwright.io_shipwrightbuilds.yaml

```diff
@@ -42,7 +42,79 @@ spec:
                 type: string
             type: object
           status:
-            description: ShipwrightBuildStatus defines the observed state of Shipwright-Build
+            description: ShipwrightBuildStatus defines the observed state of ShipwrightBuild
```

Comment:
yeah pretty sure I heard this come up in the scrum portion of our retrospective meeting today ... need to work around the word wrapping pain with controller-gen by running make generate / make bundle whatever in an ubuntu container

Reply:
that said the verify passed, and the bundle-build was skipped, in the latest github action PR verification ... so maybe we don't obsess with this like we have in shipwright-io/builds 

@adambkaplan @coreydaley @otaviof please chime in ... WDYT? .... did I in fact miss the discussion here during our last meeting?

Reply:
> yeah pretty sure I heard this come up in the scrum portion of our retrospective meeting today ... need to work around the word wrapping pain with controller-gen by running make generate / make bundle whatever in an ubuntu container

Feels like something we should fix, or invest time on it, sooner rather than later. Can be frustrating having different outcomes locally than on CI, and plus, we may reuse this effort in the Build Controller project itself later on.